pkg/apis/flows/v1/parallel_lifecycle_test.go

```diff
@@ -336,121 +327,89 @@ func TestParallelPropagateSubscriptionStatusUpdated(t *testing.T) {
 
 func TestParallelReady(t *testing.T) {
```

Comment:
In this unit test (or another one?) could we add a test that verifies that the parallel correctly sets the audiences from the subscriptions?

Reply:
Good point. For the audience, we have https://github.com/knative/eventing/blob/4951b748dfa284f32ff6e1129bf8bc02ff89c3e8/pkg/apis/flows/v1/parallel_lifecycle_test.go#L491-L496. So I guess you mean the OIDC service account names (but we have this kind of in the e2e tests: 32f59d4f0f15f04ef9f3353159d7e12fc4d98f91). But I can add a unit test tomorrow, which might help on debugging